src/ducks/transfers/TransferPage.jsx

```diff
@@ -295,6 +297,34 @@ const _Summary = ({
 
 const Summary = translate()(_Summary)
 
+const TransferSuccess = translate()(({ t, onReset, onExit }) => (
+  <div>
+    {t('transfer.success.description')}
+    <br />
+    <Button onClick={onExit} label={t('transfer.exit')} />
+    <br />
+    <Button
+      theme="secondary"
+      onClick={onReset}
+      label={t('transfer.new-transfer')}
+    />
+  </div>
+))
+
+const TransferError = translate()(({ t, onReset, onExit }) => (
+  <div>
+    {t('transfer.error.description')}
+    <br />
```

Comment:
Nit: put the text inside a `<p>` and remove the `<br />`

Reply:
Later PR will deal with modal styles, thanks !